lib/ae_mdw/aexn_contracts.ex

```diff
@@ -69,10 +69,14 @@ defmodule AeMdw.AexnContracts do
 
   def has_valid_aex141_extensions?(_extensions, _no_fcode), do: false
 
-  @spec call_meta_info(pubkey()) :: {:ok, Model.aexn_meta_info()} | :error
-  def call_meta_info(contract_pk) do
-    with {:ok, {:tuple, meta_info_tuple}} <- call_contract(contract_pk, "meta_info", []) do
-      {:ok, decode_meta_info(meta_info_tuple)}
+  @spec call_meta_info(Model.aexn_type(), pubkey()) :: {:ok, Model.aexn_meta_info()}
+  def call_meta_info(aexn_type, contract_pk) do
+    case call_contract(contract_pk, "meta_info", []) do
+      {:ok, {:tuple, meta_info_tuple}} ->
+        {:ok, decode_meta_info(meta_info_tuple)}
+
+      :error ->
+        {:ok, error_meta_info(aexn_type)}
```

Comment:
I'm having a hard time understanding the return value of this function. Perhaps we can return `{:ok, aexn_meta_info()} | :error` instead of a tuple with bogus data? The tuple `{:error, :error, nil}` has no use